### PR TITLE
change mvp to mvvm

### PR DESCRIPTION
**Pull request by deyasix**
I changed MVP to MVVM. Interface Contract, classes StatisticPresenter, StatisticModel have been removed. Class StatisticViewModel has been added. Also I did some refactoring code in StatisticFragment and in DayStatisticRepositoryImpl.

### CODE REVIEW

File: app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    private val dayStatisticRepository = DayStatisticRepositoryImpl()
+    var date: LocalDate = LocalDate.now()
+        private set
+
+    init {
+        setDayStatistic()
+    }
+
+    fun getCurrentDayStatistic(): DayStatistic? {
+        return dayStatistic.value
```
Comment (okorchysta) [inline app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt]: Why don't you use live data with an observer instead of this?

Comment (deyasix) [reply app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt]: I don't know, I was thinking about the easiest way. I can fix it. In ViewModel:
`fun getDayStatistic() : LiveData<DayStatistic> {
        return dayStatistic
    }`
In Fragment: 
`viewModel.getDayStatistic().observe(viewLifecycleOwner) {
            setDayStatistic(it)
        }`
Is it correct?

Comment (okorchysta) [reply app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt]: the usual definition is: 
```
private val _dayStatistic = MutableLiveData<DayStatistic?>()
val dayStatistic: LiveData<DayStatistic?>
        get() = _dayStatistic
```

Comment (deyasix) [reply app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt]: Thanks. I understood. Is there something else wrong?

Comment (okorchysta) [reply app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt]: I think no. But please, add these changes

**Review by okorchysta — APPROVED** [review]

**Pull request merged**